pandas/tests/frame/test_quantile.py

```diff
@@ -467,3 +467,17 @@ def test_quantile_empty(self):
 
         # FIXME (gives NaNs instead of NaT in 0.18.1 or 0.19.0)
         # res = df.quantile(0.5, numeric_only=False)
+
+    def test_quantile_empty_no_columns(self):
```

Comment:
might be worth testing this with timedelta & period data as well (just to confirm that they work), but can do followup.